CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+
+* We talk about the language, its related libraries and infrastructure, and our experiences using it.
+* We help PureScript programmers learn the language, and we ask for help becoming stronger programmers ourselves. 
+* We discuss general functional programming concepts and other parts of programming, type theory, and mathematics that are relevant to Purescript.
+* We organize open-source work, meetups, hackathons, and other community events.
+* We discuss what we can do to grow an inclusive, friendly, respectful community.
+
+## What is okay?
+
+Examples of behavior that contributes to a positive environment for our community include:
+
+* Demonstrating empathy and kindness toward other people.
+* Being respectful of differing opinions, viewpoints, and experiences.
+* Giving and gracefully accepting constructive feedback.
+* Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience.
+
+As participants in the PureScript community, we pledge to make participation in our community a harassment-free experience for everyone -- regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, caste, color, religion, or sexual identity and orientation.
+
+## What is not okay?
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of any kind.
+* Trolling, insulting or derogatory comments, and personal or political attacks.
+* Public or private harassment.
+* Publishing others’ private information, such as a physical or email address, without their explicit permission.
+* Other conduct which could reasonably be considered inappropriate in a professional setting.
+
+
+## Where does this Code of Conduct apply?
+
+This code of conduct covers anyone participating in PureScript community channels, including our Discord server, Discourse forum, and GitHub properties. For members of the PureScript team, it applies anywhere they are representing the PureScript project (for example, on social media).
+
+## How do members report violations?
+
+You can report abusive, harassing, or otherwise unacceptable behavior to the community moderators team or to a particular moderator if you are not comfortable sharing your concern with the full team.
+
+All complaints will be reviewed and investigated promptly and fairly. Your communication with moderators will not be shared with anyone else without your permission.
+
+Here are per-platform instructions:
+
+* Discord: Send a private message to the `@Moderators` team. To find a specific moderator to message, use the members sidebar under the `Moderators` role to find their profile.
+* Discourse: Send a private message to the `moderators` group. To find a specific moderator to message, see the `moderators` group page: https://discourse.purescript.org/g/moderators
```

Comment:
This group page doesn't exist or is private, says Discourse.

Reply:
It works for me, so it must only be visible to moderators. This is part of why I would prefer we list moderators explicitly with their names and email addresses here - I think making it easy to report incidents should be the priority, and listing email addresses seems like the easiest and most foolproof way to ensure that people can report incidents. Requiring people to navigate or sign up for some service they're not familiar with or for which the instructions might not work could be a very poor experience if you're already upset because of the incident you're trying to report, and not everyone is on Discord or Discourse.

Reply:
I did find that moderators can be found directly via the `about` page, which applies for everyone regardless of being logged in:
https://discourse.purescript.org/about